### PR TITLE
#162727487 comment update history

### PR DESCRIPTION
**Pull request by julietezekwe**
#### What does this PR do?
This PR implements comment edit history track

#### Description of Task to be completed?
Comment edit history can be saved and be retrieved

#### How should this be manually tested?
On the update comment route  PUT `/api/v1/arts/comments/commentId`
Update a comment with a certain  more as many times as you want
Hit GET `/api/v1/arts/coments/commentId`  with same ID and get all the edit history

#### Any background context you want to provide?
Run `npm run db:seed` before testing

#### What are the relevant pivotal tracker stories?
[#162727487](https://www.pivotaltracker.com/story/show/162727487)

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

File: src/controllers/CommentsController.js [hunk 1]
```diff
@@ -52,6 +52,105 @@ class CommentsController {
...
+   * @param {Object} req
+   * @param {object} res
+   * @return {object} updated comment
+   */
+  static async updateComment(req, res) {
+    try {
+      const { body } = req.body;
+      const { commentId } = req.params;
+      const { id } = req.verifyUser;
+      const findComment = await Comment.find({
```
Comment (andela-dbamidele) [inline src/controllers/CommentsController.js]: Can we move this to the middleware. I think it will make more sense there.

**Review by andela-dbamidele — APPROVED** [review]

File: src/controllers/CommentsController.js (hunk 1 above)
Comment (andela-dbamidele) [reply src/controllers/CommentsController.js]: You didn't fix this....

Comment (julietezekwe) [reply src/controllers/CommentsController.js]: Oh just seeing this. will do now

Comment (julietezekwe) [reply src/controllers/CommentsController.js]: Done

**Pull request merged**